package.json

```diff
@@ -3,36 +3,21 @@
   "version": "0.0.1",
   "description": "",
   "main": "build/index.js",
-  "author": "Luc Succes <l2succes@gmail.com> & Art.sy Inc",
+  "authors": [
+    "Luc Succes <l2succes@gmail.com>",
+    "Art.sy Inc"
+  ],
   "license": "MIT",
-  "devDependencies": {
-    "@types/jest": "^23.3.2",
-    "@types/lodash": "^4.14.116",
-    "@types/mongodb-uri": "^0.9.0",
-    "@types/morgan": "^1.7.35",
-    "@types/node": "^10.11.0",
-    "danger": "^4.0.2",
-    "dotenv": "^6.1.0",
-    "husky": "^1.1.2",
-    "jest": "^23.6.0",
-    "lint-staged": "^7.3.0",
-    "prettier": "^1.14.3",
-    "release-it": "^7.6.1",
-    "ts-jest": "^23.10.1",
-    "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
-    "typescript": "^3.0.3"
-  },
   "scripts": {
-    "type-check": "tsc --noEmit",
+    "bootstrap-data": "ts-node scripts/bootstrapData.ts",
```

Comment:
Added `ts-node` so this would be executable without needing to be compiled first. Also moved app-level `scripts` stuff top level and out of sources.